Datadog.Trace.sln

```diff
@@ -526,6 +526,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Samples.Probes.Unreferenced
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Sandbox.LegacySecurityPolicy", "tracer\test\test-applications\regression\Sandbox.LegacySecurityPolicy\Sandbox.LegacySecurityPolicy.csproj", "{959E9599-8D99-43BC-8038-B91F76179C1C}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Datadog.AutoInstrumentation.Generator", "tracer\src\Datadog.AutoInstrumentation.Generator\Datadog.AutoInstrumentation.Generator.csproj", "{95DEBF4E-6017-40AD-9840-C7D112591228}"
+EndProject
```

Comment:
The downside to adding it to the solution here is that means that the avalonia packages will be restored and copied around on every build in CI. I think it would probably be preferable to not include the project in the solution, especially as it's only meant to be run from Nuke locally?
```suggestion
```

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/4422/commits/01fcf69054a323668f0be7f36bed0fcfef6943ac